upring.js

```diff
@@ -76,6 +76,29 @@ function UpRing (opts) {
   }
 
   this._peers = {}
+
+  var that = this
+  this.onClose(function (_err, cb) {
```

Comment:
If you use three parameters you can access the instance without the need to create a closure:
```js
this.onClose(function(_err, istance, cb) {
```